ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.14.yaml

```diff
@@ -78,7 +78,9 @@ tests:
   cron: '@yearly'
   steps:
     cluster_profile: ibmcloud-qe
-    workflow: cucushift-installer-rehearse-ibmcloud-ipi-proxy-private
+    env:
+      RT_ENABLED: "true"
+    workflow: cucushift-installer-rehearse-ibmcloud-ipi
```

Comment:
If we want to reuse the existing ci jobs, pls combine the `RT_ENABLED` coverage on an existing ci jobs.